x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/routes/time_series_query.ts

```diff
@@ -33,21 +33,15 @@ export function createTimeSeriesQueryRoute(service: Service, router: IRouter, ba
     req: KibanaRequest<any, any, TimeSeriesQuery, any>,
     res: KibanaResponseFactory
   ): Promise<IKibanaResponse> {
-    service.logger.debug(`route query_data request: ${JSON.stringify(req.body)}`);
+    service.logger.debug(`route ${path} request: ${JSON.stringify(req.body)}`);
 
-    let result: TimeSeriesResult;
-    try {
```

Comment:
try/catch was removed, as we now catch the only directly throwable API (above), so anything else would be a JS runtime error.